src/vs/platform/accessibilitySignal/browser/accessibilitySignalService.ts

```diff
@@ -359,7 +359,7 @@ export const enum AccessibilityAlertSettingId {
 	OnDebugBreak = 'accessibility.alert.onDebugBreak',
 	NoInlayHints = 'accessibility.alert.noInlayHints',
 	LineHasBreakpoint = 'accessibility.alert.lineHasBreakpoint',
-	ChatResponsePending = 'accessibility.alert.chatResponsePending'
+	Progress = 'accessibility.alert.chatResponseProgress'
```

Comment:
this is a deprecated setting not worth updating, will be removed soon https://github.com/microsoft/vscode/issues/210658